### DISCUSSION TITLE
uriString

### DISCUSSION THREAD

**Discussion by beng7** [bitfireAT/icsx5#182] [General]
Hi 
I simply get this error message after I input a name for a subscription (https://raw.githubusercontent.com/PanderMusubi/lunar-phase-calendar/master/fr/full-moon.ics ; the link was provided to me by its creator) and try to save it.
At least is it possible to have a clearer message so we know what the issue is?
Thanks

**Comment by ArnyminerZ** [comment]
For me it's working fine. Are you sure that you are copying the link correctly?

**Comment by beng7** [comment]
Hi Arnau 

Indeed, I've tried so may times, copied then pasted it into a plain text editor then copied it from the text editor to ICSx to the same effect :( 
thanks for your help 
b 

----- Original Message -----

**Reply by ArnyminerZ** [reply]
Can you access it through a normal browser? It should show some plain text. If you can't access it through a generic web browser (eg Chrome) you have something misconfigured in your device.
